go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/friendsofgo/killgrave
 
-go 1.13
+go 1.15
```

Comment:
why this change? we're introducing  something that's is only working on go 1.15.

Furthermore you should updgrade the `.circleci/config.yml` to build and testing on 1.14 and 1.15

Reply:
Okay, thanks for the suggestion! 🙏  I'm gonna update the Go versions defined on the CI configuration 🚀